streaming/src/test/scala/org/apache/spark/streaming/scheduler/ExecutorAllocationManagerSuite.scala

```diff
@@ -394,3 +398,20 @@ class ExecutorAllocationManagerSuite extends SparkFunSuite
     }
   }
 }
+
+class StreamManualClock(time: Long = 0L) extends ManualClock(time) with Serializable {
```

Comment:
can you write some high level classdoc on how this is different from ManualClock? Otherwise 2 years from now we won't remember what this is about.